File: OrderService/Models/OrderItem.cs
using System;

namespace OrderService.Models
{
    public class OrderItem
    {
        public OrderItem()
        {
            
        }
        public OrderItem(int id, String name, int productCategory, decimal price, int quantity)
        {
            Id = id;
            Name = name;
            ProductCategory = productCategory;
            Price = price;
            Quantity = quantity;
        }
        public int Id { get; set; }
        public int Quantity { get; set; }
        public String Name { get; set; }
        public decimal Price { get; set; }
        public int ProductCategory { get; set; }
      
    }
}

File: AngularNetCoreProj/ClientApp/src/app/entity/OrderItem.ts

export class IOrderItem {
  constructor(id, name, orderId, quantity, price, productCategory) {
    this.id = id;
    this.name = name;
    this.orderId = orderId;
    this.quantity = quantity;
    this.price = price;
    this.productCategory = productCategory;
  }
    orderId: number;
    quantity: number;
    name: string;
    price: number;
    productCategory: number;
    id?: number;
}
